src/inmanta/types.py

```diff
@@ -60,6 +62,11 @@ def issubclass(sub: type, super: Union[type, tuple[type, ...]]) -> bool:
 
 JsonType = dict[str, Any]
 ReturnTupple = tuple[int, Optional[JsonType]]
+StrictJson = TypeAliasType(
+    "StrictJson",
+    "Union[dict[str, StrictJson], list[StrictJson], str, int, float, bool, None]",
+)
+
```

Comment:
I think this is equivalent to `type StrictJson = dict[str, StrictJson] | list[StrictJson] | str | int | float | bool | None`, and that `TypeAliasType` is discouraged from direct use.

Reply:
thx!

Reply:
This is still the same. Forgot to push?